usmqe_tests/alerting/test_email_common.py

```diff
@@ -0,0 +1,30 @@
+"""
+Alerting test suite - general tests
+"""
+import pytest
+import usmqe.usmmail
+from datetime import datetime
+
+
+LOGGER = pytest.get_logger('alerting_common', module=True)
+
+
+def test_no_messages_from_future():
+    """
+    Alerting: common
+    *******************
+
+    .. test_metadata:: author ebondare@redhat.com
+
+    Description
+    ===========
+
+    Test that there are no messages with date later than the current date
+    """
+    messages = usmqe.usmmail.get_msgs_by_time(start_timestamp=datetime.now().timestamp())
```

Comment:
We can't expect that there will be any messages in mailbox at this point (no fixture which would trigger alerts or previous steps in the test case). But when no email has been send so far, this test will fail (file `/var/mail/root` doesn't exist).